weblog/server.go

```diff
@@ -23,9 +23,9 @@ type Server struct {
 }
 
 // NewServer returns a pointer to a new Server instance.
-func NewServer(syslogishServer *syslogish.Server) (*Server, error) {
+func NewServer(storageAdapter storage.Adapter) (*Server, error) {
```

Comment:
This should be a pointer.  See https://github.com/deis/logger/pull/85/files#r67026999.